nginx.conf

```diff
@@ -0,0 +1,43 @@
+
+error_log  logs/error.log  debug;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
```

Comment:
Is `65` the default value?

Reply:
this comes from the nginx conf, we just added datadome config and and proxy_pass to a web page.